### DISCUSSION TITLE
How to wrap a c_char as vlang's new basic data type 'char'?

### DISCUSSION THREAD

**Discussion by theseusyang** [vlang/v#13548] [Ideas]
Hi team,
How to wrap a c_char as vlang new basic data type 'char'? or What API need to be invoked in vlang compiler?

**Comment by spytheman** [comment]
V's `char` is the same as C's `char`

**Reply by spytheman** [reply]
you can cast values to it with `x := char(1)` - it will generate `char x = ((char)(1));`
